lms/djangoapps/bulk_email/models.py

```diff
@@ -170,10 +170,10 @@ def ensure_valid_cohort(cls, cohort_name, course_id):
             cohort = get_cohort_by_name(name=cohort_name, course_key=course_id)
         except CourseUserGroup.DoesNotExist:
             raise ValueError(
-                "Cohort {cohort} does not exist in course {course_id}".format(
+                u"Cohort {cohort} does not exist in course {course_id}".format(
                     cohort=cohort_name,
                     course_id=course_id
-                )
+                ).encode('utf8')
```

Comment:
Why did you encode this? Does `ValueError` not accept unicode objects?

Reply:
discussed in slack